Content.Server/SS220/Commands/AdminRandomMakeAntagSystem.cs

```diff
@@ -0,0 +1,126 @@
+// © SS220, An EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt
+using System.Linq;
+using Content.Server.Administration;
+using Content.Server.Antag;
+using Content.Server.GameTicking;
+using Content.Server.GameTicking.Rules.Components;
+using Content.Shared.Administration;
+using Content.Shared.GameTicking;
+using Content.Shared.IdentityManagement;
+using Robust.Server.Player;
+using Robust.Shared.Console;
+using Robust.Shared.Random;
+using Content.Shared.Ghost;
+using Content.Server.Administration.Managers;
+using Content.Server.Roles;
+using Content.Server.Roles.Jobs;
+using Content.Server.Preferences.Managers;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+using Content.Server.Antag.Components;
+using Robust.Shared.Utility;
+using Robust.Server.GameObjects;
+using Content.Shared.Mind;
+using Content.Shared.Mindshield.Components;
+
+namespace Content.Server.SS220.Commands
+{
+    [AdminCommand(AdminFlags.Ban)] // Only for admins
+    public sealed class MakeAntagCommand : IConsoleCommand
+    {
+        [Dependency] private readonly IEntityManager _entityManager = default!;
+
+        public string Command => "makerandomantag";
+        public string Description => "Делает случайного игрока антагонистом из предложенного списка с учетом джобок.";
+        public string Help => $"Usage: {Command}";
+
+        private readonly List<string> _antagTypes = new() // TODO: When will add a cult add a cultist there
+        {
+            "Traitor",
+            "Thief",
+            "InitialInfected",
+        };
+
+        public void Execute(IConsoleShell shell, string argStr, string[] args)
+        {
+            if (args.Length == 0 || !_antagTypes.Contains(args[0]))
+            {
+                shell.WriteLine("Необходимо указать тип антагониста: Traitor, Thief, или InitialInfected.");
+                return;
+            }
+
+            var isSuccessEntityUid = AdminMakeRandomAntagCommand(args[0]);
+
+            if (isSuccessEntityUid != null)
+            {
+                shell.WriteLine($"{Identity.Name(isSuccessEntityUid.Value, _entityManager)} успешно стал {args[0]}");
+            }
+            else
+                shell.WriteLine($"Никто не стал антагонистом, потому что все итак антагонисты! Ну или в джоббане.. Ну или с МЩ.. Ну или сами разбирайтесь, как оно работает!");
+        }
+
+        private EntityUid? AdminMakeRandomAntagCommand(string defaultRule)
+        {
+            var antag = _entityManager.System<AntagSelectionSystem>();
+            var playerManager = IoCManager.Resolve<IPlayerManager>();
+            var gameTicker = _entityManager.System<GameTicker>();
+            var robust = IoCManager.Resolve<IRobustRandom>();
```

Comment:
нейминг вышел из чата
```suggestion
            var random = IoCManager.Resolve<IRobustRandom>();
```